tests/test_processors.py

```diff
@@ -174,3 +176,54 @@ def brand(self):
 )
 def test_format_price(input_value, expected_value):
     assert _format_price(Price.fromstring(input_value)) == expected_value
+
+
+gtin_str = "978-1-933624-34-1"
+gtin_html = f"<span>{gtin_str}</span>"
+gtin_expected = [Gtin("isbn13", "9781933624341")]
+
+
+@pytest.mark.parametrize(
+    "input_value,expected_value",
+    [
+        (None, None),
+        ([], None),
+        ("foo", None),
+        (Selector(text="<html></html>"), None),
+        (SelectorList([]), None),
```

Comment:
One last thing: could you please add a test for the case SelectorList matches several gtins?